packages/sources/tiingo/src/config.ts

```diff
@@ -5,7 +5,7 @@ export const NAME = 'TIINGO'
 
 export const DEFAULT_ENDPOINT = 'crypto'
 export const DEFAULT_BASE_URL = 'https://api.tiingo.com'
-export const DEFAULT_WS_API_ENDPOINT = 'wss://api.tiingo.com/crypto'
```

Comment:
We should extend our framework in the future to allow supporting multiple WS endpoints.  This should be ok for now as we want to disable WS for `crypto` anyways.